fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -162,7 +165,7 @@ def build_upload_command(username, password, source = "/tmp", provider_short_nam
         "-u \"#{username}\"",
         "-p #{shell_escaped_password(password)}",
         "-f \"#{source}\"",
-        ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"], # that's here, because the user might overwrite the -t option
+        additional_upload_parameters, # that's here, because the user might overwrite the -t option
         "-t Signiant",
```

Comment:
This will give two `-t` parameters. Is that going to mess things up? 🤔

Reply:
No, the first one should take precedence (so the second one does _not_ overwrite the first one)